packages/devel/libevent/package.mk

```
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libevent"
+PKG_VERSION="2.1.12-stable"
+PKG_SHA256="92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb"
+PKG_LICENSE="BSD-3-Clause"
```

```suggestion
PKG_LICENSE="BSD-3-Clause-Modification"
```

I guess ? [as the 3-clause (or "modified") BSD license:](https://github.com/libevent/libevent/blob/master/LICENSE#L2)

---

BSD-3-Clause seems to be the correct one https://spdx.org/licenses/BSD-3-Clause.html, the -Modification is actually a 4-llause license